src/core/library.c

```diff
@@ -778,6 +778,40 @@ QuicLibrarySetGlobalParam(
         break;
 #endif
 
+#ifdef DEBUG
+    case QUIC_PARAM_GLOBAL_ALLOC_FAIL_DENOMINATOR: {
+        if (BufferLength != sizeof(int32_t)) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+        int32_t Value = *(int32_t*)Buffer;
+        if (Value < 0) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+        CxPlatSetAllocFailDenominator(Value);
+
```

Comment:
nit: either remove this empty newline or add one before `int32_t Value = ...`. Same with below.